qiskit/tools/qi/pauli.py

```diff
@@ -102,27 +102,49 @@ def to_matrix(self):
         Order is q_n x q_{n-1} .... q_0
         """
         x = np.array([[0, 1], [1, 0]], dtype=complex)
+        y = np.array([[0, -1j], [1j, 0]], dtype=complex)
         z = np.array([[1, 0], [0, -1]], dtype=complex)
         id_ = np.array([[1, 0], [0, 1]], dtype=complex)
-        x_temp = 1
+        matrix = 1
         for k in range(self.numberofqubits):
-            if self.v[k] == 0:
-                temp_z = id_
-            elif self.v[k] == 1:
-                temp_z = z
+            if self.v[k] == 0 and self.w[k] == 0:
+                new = id_
+            elif self.v[k] == 1 and self.w[k] == 0:
+                new = z
+            elif self.v[k] == 0 and self.w[k] == 1:
+                new = x
+            elif self.v[k] == 1 and self.w[k] == 1:
+                new = y
             else:
-                print('the z string is not of the form 0 and 1')
-            if self.w[k] == 0:
-                temp_x = id_
-            elif self.w[k] == 1:
-                temp_x = x
+                print('the string is not of the form 0 and 1')
+            matrix = np.kron(new, matrix)
+
+        return matrix
+
+    def to_spmatrix(self):
+        """Convert Pauli to a sparse matrix representation (CSR format).
+
+        Order is q_n x q_{n-1} .... q_0
+        """
+        x   = sparse.csr_matrix(np.array([[0, 1], [1, 0]], dtype=complex))
```

Comment:
Can you fix the code style issues ([as shown in lines 583-592 of the CI job](https://travis-ci.org/QISKit/qiskit-sdk-py/jobs/347899478))? They are very minor and due to the extra spaces, you can run `make lint` on your local machine afterwards to verify they are passing.